packages/client/src/websocket.ts

```diff
@@ -189,9 +189,12 @@ export default class WebSocketClient {
                         console.log('long timeout, or server restart, or sequence number is not found.'); //eslint-disable-line no-console
 
                         this.missedEventCallback?.();
-                        this.missedMessageListeners.forEach((listener) => listener());
 
-                        this.serverSequence = 0;
+                        try {
+                            this.missedMessageListeners.forEach((listener) => listener());
+                        } finally {
```

Comment:
Instead of using a try-finally here, I think we might want to wrap each listener call in a try-catch so that we still run any listeners after the failing one and so that we finish the rest of `onmessage` after an error happens. So it'd be more like
```js
for (const listener of this.missedMessageListeners) {
    try {
        listener();
    } catch (e) {
        // log the error or something
    }
}
```

Reply:
👍 Changed with logging a descriptive message to identify easily the listener that failed